scripts/bump_lsp_version.js

```diff
@@ -0,0 +1,12 @@
+import { bumpExtensionVersionInScriptFiles } from './bump_extension_version'
+
+
+if (require.main === module) {
+  const args = process.argv.slice(2)
+  bumpExtensionVersionInScriptFiles({
+    nextVersion: args[0],
+    branch_channel: args[1],
+    extensionBump: false
+  })
+  return version
```

Comment:
It is not a function, so this return is probably incorrect?

Reply:
this function is not needed anymore anyway, I deleted it 👍